iron-form.html

```diff
@@ -230,8 +253,13 @@
     },
 
     _registerElement: function(e) {
-      e.target._parentForm = this;
-      this._customElements.push(e.target);
+      var element = e.target;
+      element._parentForm = this;
+      this._customElements.push(element);
+
+      // Save the original value of this input.
+      this._customElementsInitialValues.push(
+          this._usesCheckedInsteadOfValue(element) ? element.checked : element.value);
```

Comment:
I just saw the one other use below.. I'm going to downgrade this comment to :bike: :house: